src/PKSim.Presentation/DTO/Mappers/ExpressionParametersToSimulationExpressionsDTOMapper.cs

```diff
@@ -124,7 +124,7 @@ private ExpressionContainerDTO expressionContainerFor(IParameter relativeExpress
 
       private bool moleculeIsTransporter(IndividualMolecule molecule)
```

Comment:
There is no way to find (I believe) from the Global Molecule container in the model, that a container represents a transporter etc..

```
var container  = model.root.GetChildrenByName("CYP3A4")
container.MoleculeType == ENZYME <== that we do not have
```

At the moment, I am returning isTransporter false. But that would be better if we could actually retrieve the info from the container itself